### PR TITLE
Add `debugError` diff tests

### PR DESCRIPTION
**Pull request by noisersup**
Closes FerretDB/FerretDB#640.

This PR introduces tests for a new `debugError` command, but it also temporary fixes one failing test that was pushed to the main branch because of the recent bug that made dance flaky.

### CODE REVIEW

File: tests/diff/debug_error_test.go [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+)
+
+func TestDebugError(t *testing.T) {
+	t.Parallel()
+
+	for name, tc := range map[string]struct {
+		input       string
+		expectedErr *mongo.CommandError
+		expectedRes bson.D
+	}{
```
[the remark this answers is not in the document]
Comment (rumyantseva) [reply tests/diff/debug_error_test.go:32]: Nice catch!

**Review by rumyantseva — CHANGES_REQUESTED** [review]
As @chilagrow mentioned, the case with `panic` is missing. The rest looks good to me!

File: tests/diff/debug_error_test.go [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+		},
+		"string": {
+			input: "foo",
+			expectedErr: &mongo.CommandError{
+				Code:    1,
+				Name:    "InternalError",
+				Message: "foo",
+			},
+		},
+	} {
```
Comment (rumyantseva) [inline tests/diff/debug_error_test.go:69]: Maybe also one more case for empty input:

```go
input: ""
```

**Review by w84thesun — COMMENTED** [review]
LGTM, except for those mentioned cases.

File: tests/diff/debug_error_test.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (noisersup) [reply tests/diff/debug_error_test.go:32]: done

**Review by rumyantseva — APPROVED** [review]
LGTM

**Review by chilagrow — APPROVED** [review]

File: tests/diff/debug_error_test.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (chilagrow) [reply tests/diff/debug_error_test.go:32]: Thanks for the detailed comment :pray:

**Review by w84thesun — APPROVED** [review]

**Pull request merged**